Rubberduck.sln

```diff
@@ -485,8 +485,8 @@ Global
 		{D488071E-EDCB-4601-B734-1A3109ED903C}.Release64|x64.Build.0 = Release|Any CPU
 		{D488071E-EDCB-4601-B734-1A3109ED903C}.Release64|x86.ActiveCfg = Release|Any CPU
 		{D488071E-EDCB-4601-B734-1A3109ED903C}.Release64|x86.Build.0 = Release|Any CPU
-		{F83B6746-49A6-4CFD-9A29-3D7BBD4F0323}.Debug|Any CPU.ActiveCfg = Release|Any CPU
-		{F83B6746-49A6-4CFD-9A29-3D7BBD4F0323}.Debug|Any CPU.Build.0 = Release|Any CPU
+		{F83B6746-49A6-4CFD-9A29-3D7BBD4F0323}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
Is there a reason for this config change?

Reply:
I have no idea why that changed. It shouldn't have.